App.js

```diff
@@ -1,18 +1,17 @@
+import 'react-native-gesture-handler';
 import * as React from 'react';
-import {NavigationContainer} from '@react-navigation/native';
-import {createStackNavigator} from '@react-navigation/stack';
-import Login from './Login.js'
+import { View, Text } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import Login from './Components/Login.js';
+
 const Stack = createStackNavigator();
 
-function MyStack() {
+export default function MyStack() {
   return (
     <NavigationContainer>
       <Stack.Navigator>
-        <Stack.Screen
-          name='Login'
-          component={ Login }
-          options={{title: 'Welcome'}}
-        />
+        <Stack.Screen name="Home" component={ Login } />
```

Comment:
So is this what allows it to navigate to the login screen?

Reply:
Yes , and even `home` shows up on the page as a label. very cool